test/filter.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const assert = require('assert'),
+      filter = require('../lib/filter');
+
+function nullLog() {
+    // Do nothing. Log messages could be verified but aren't.
+}
+
+suite('Event filtering', function() {
+    test('null body', function() {
+        assert.strictEqual(filter.event(null, nullLog), false);
+    });
+    test('empty body', function() {
```

Comment:
In the application code, the word "empty" is used to describe a falsey value. In the tests, it's used to describe an object with no "own" properties. I'd support either one, but using it for both is confusing.

Reply:
I went with "no body" in `filterEvent`.